tests/cli/test_cli.py

```diff
@@ -2366,6 +2366,22 @@ def test_nonexistent(ctx):
     _run_command([cl, 'work', 'nonexistent::'], expected_exit_code=1)
 
 
+@TestModule.register('worker_manager')
+def test_false_login(ctx):
+    username = os.getenv("CODALAB_USERNAME")
+    password = os.getenv("CODALAB_PASSWORD")
+    del os.environ["CODALAB_USERNAME"]
+    del os.environ["CODALAB_PASSWORD"]
+    _run_command([cl, 'logout'])
+    os.environ["CODALAB_USERNAME"] = username
+    os.environ["CODALAB_PASSWORD"] = "wrongpassword"
+    result = _run_command([cl_worker_manager, '--server=https://worksheets.codalab.org/', 'slurm-batch', '--partition', 'foo'], 1)
+    print("result is " + str(result))
+    sys.stdout.flush()
+    os.environ["CODALAB_USERNAME"] = username
```

Comment:
Isn't this already handled inside the cleanup logic?

Reply:
I checked the example in `test_auth`

Reply:
Why is the test for this PR specifically targeting `cl-worker-manager`? it might be simpler to just add a new test to `test_auth` with the regular `cl` command.

Reply:
Also, we need to have a test that checks to ensure that "Your login info is incorrect. Please try again." is printed to the console when the user enters incorrect credentials.